.github/workflows/release-airgap-artifacts.yaml

```diff
@@ -0,0 +1,47 @@
+name: Release airgap artifacts
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  release-airgap-artifacts:
+    name: Start airgap release
+    runs-on:
+      - self-hosted
+      - medium
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY }}
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
```

Comment:
does this work for tags as well? 

in other repos I've done something like this (we can either use output or envs, either way but i think the ref ends up being `refs/tags/` and not `refs/heads/`
```
echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```